### DISCUSSION TITLE
Welcome to geograpy3 Discussions!

### DISCUSSION THREAD

**Discussion by somnathrakshit** [somnathrakshit/geograpy3#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WolfgangFahl** [comment]
is this a new feature of github?

**Reply by somnathrakshit** [reply]
Yes, this was launched within the last few months.

**Comment by somnathrakshit** [comment]
Hello @WolfgangFahl, let's start using this forum. I saw your projects that you mentioned on Gitter. I will be happy to help you in creating a flask/bootstrap based service. Can you meet sometime this week?

**Reply by WolfgangFahl** [reply]
Yes sure and Tim and Musaab would also be interested to join and learn to know you. How can we get the participants of the gitter chat moved over here?

**Reply by WolfgangFahl** [reply]
In what timezone are you right now? It's currently noon in my time zone and I am available until about 6 pm maybe even 8pm today and things look also good in the upcoming days.

**Reply by WolfgangFahl** [reply]
see https://github.com/somnathrakshit/geograpy3/issues/29
